test/lib/ufe/testGroupCmd.py

```diff
@@ -179,10 +184,7 @@ def testUsdGroup(self):
 
         # gloabl selection should not be empty after undo.
         if (os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') > '3004'):
-            pass
-            # TODO: for some strange reasons the globalSelection returns 0??
-            # I tried the same steps in interactive Maya and it correctly returns 2
-            # self.assertEqual(len(globalSelection), 2)
+            self.assertEqual(len(globalSelection), 2)
```

Comment:
@ppt-adsk  Ah, I see why the selection reported 0 after undo. It is because nothing was selected in the first place when the group command ran. :)  Should be good now.